wallet-api/src/Wallet/Visualisation.hs

```diff
@@ -0,0 +1,101 @@
+{-# LANGUAGE DeriveAnyClass       #-}
```

Comment:
Does @j-mueller actually want this committed?

Reply:
Good point, and done.